File: src/main/java/rusha/x/Id.java
package rusha.x;

import java.util.Scanner;

public class Id {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] usedIdIndexes = new int[n];

        for (int i = 0; i < n - 2; i++) {
            int id = scanner.nextInt();
            usedIdIndexes[id - 1] = 1;
        }

        for (int i = 0; i < n; i++) {
            if (usedIdIndexes[i] == 0) {
                System.out.print(i + 1);
                System.out.print(" ");
            }
        }
    }
}

File: src/main/kotlin/rusha/x/Id.kt
package rusha.x

import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    val ids: MutableList<Int> = mutableListOf()
    val n: Int = scanner.nextInt()

    repeat(n - 2) {
        val id = scanner.nextInt()
        ids.add(id)
    }

    val result = mutableListOf<Int>()

    (1..n).forEach { i ->
        if (!ids.contains(i)) {
            result.add(i)
        }
    }
    println(result.joinToString(separator = " "))
}

